backend/migrating/v2/constants.py

```diff
@@ -0,0 +1,2 @@
+class V2:
+    SCHEMA_NAME = "unstract_v2"
```

Comment:
NIT: Post migration will we be able to rename this? Do you think it'll be good to leave this as `unstract_v2` in the long run? If not, we could just go with `unstract`
cc: @muhammad-ali-e @hari-kuriakose @ritwik-g

Reply:
I think we can revert back it to env and make the variable name as DB_SCHEMA as mentioned here: https://github.com/Zipstack/unstract/pull/538#pullrequestreview-2273753650